test/test_math_ops.cpp

```diff
@@ -1614,6 +1614,28 @@ TEST(MathOps, Log2D) {
   opexecuter.RunTest();
 }  // end of test op Log
 
+TEST(MathOps, Log1p) {
+  Scope root = Scope::NewRootScope();
+  int dim1 = 2;
+  int dim2 = 3;
+  vector<float> vals = {0, 0.25, 0.5, 1, 5, 10};
```

Comment:
can u add some more numbers which have negative numbers in between -1 to 0

and if the input is < -1? how does TF behave and are we doing the same behaviour when running through ng?

Reply:
@sayantan-nervana 
I tried to add the negative value to the test, i.e., -2, -1, both ngraph and TF have the same result but the test case fails as shown below. The reason is that the comparison in test_utilities.h uses _arg0 == arg1_, this won't work for nan and inf. So, can we just keep the current test data set?
`/home/yimeisun/ngraph-bridge/test/test_utilities.h:183: Failure
Value of: rt
  Actual: false
Expected: true
 TF output -nan
 NG output -nan
/home/yimeisun/ngraph-bridge/test/test_utilities.h:183: Failure
Value of: rt
  Actual: false
Expected: true
 TF output -inf
 NG output -inf
`

Reply:
@noim210 was facing the same issue i think, where the equality does not work for inf/nan. Can you plz fix the comparing function as well

Reply:
Done in PR491.